src/components/icons.js

```diff
@@ -0,0 +1,72 @@
+import {Box, Heading, Link, TextInput} from '@primer/react'
+import {SearchIcon} from '@primer/octicons-react'
+import {Link as GatsbyLink} from 'gatsby'
+import React from 'react'
+import icons from '@primer/octicons-react/build/data.json'
+import useSearch from './use-search'
+import Icon from './icon'
+
+export default function Icons() {
+  const [query, setQuery] = React.useState('')
+  const iconsArray = React.useMemo(() => {
+    return Object.values(icons).flatMap(icon => Object.entries(icon.heights).map(([height, value]) => ({
+        name: icon.name,
+        keywords: icon.keywords,
+        width: value.width,
+        height,
+        path: value.path
+      }))
+    )
+  }, [icons])
+  const results = useSearch(iconsArray, query, {keys: ['name', 'keywords']})
+  // group icons by height property
+  const iconsByHeight = React.useMemo(() => results.reduce((acc, cur) => {
+    // create height array if it doesn't exist
+    if (!acc[cur.height]) acc[cur.height] = []
+    // add current icon to height array
+    acc[cur.height].push(cur)
+
+    return acc
+  }, {}), [results])
+
+  return (
+    <Box display={"grid"} gridGap={5}>
+      <TextInput
+        icon={SearchIcon}
```

Comment:
For some reason the search icon placement is looking broken, is this a known issue?
<img width="238" alt="Screen Shot 2023-02-07 at 12 08 23 PM" src="https://user-images.githubusercontent.com/586552/217314893-8ee6f90e-d7d6-467d-804c-19d9564f2aa8.png">

Reply:
Fixed ✅